tests/test_bad_coords.py

```diff
@@ -0,0 +1,83 @@
+import pprint
+
+from scrapy import Spider
+from scrapy.crawler import Crawler
+
+from locations.items import Feature
+from locations.pipelines.check_item_properties import CheckItemPropertiesPipeline
+
+
+def get_objects(lat, lon):
+    spider = Spider("test")
+    spider.crawler = Crawler(Spider)
+    return (
+        [
+            Feature(lat=lat, lon=lon),
+            Feature(
+                geometry={
+                    "type": "Point",
+                    "coordinates": [lat, lon],
```

Comment:
GeoJSON does [x, y], so it should be [lon, lat]

```suggestion
                    "coordinates": [lon, lat],
```